### PR TITLE
Made cat scalable

### LINKED ISSUE #273: scale the cat with window size
### Role

I like cats.

### Suggestion

Scale the cat with the size of the window

### Benefit

So the cat will always be visible

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

### You may use the editor below to elaborate further.

As of right now the cat doesn't scale with the size of the window:
image

The cat should become smaller if it doesn't fit into the window, or become bigger to fill the space so there isn't just an empty void.
image

### PR DESCRIPTION
**Pull request by Trial97**
Fixes #273

### CODE REVIEW

**Review by getchoo — APPROVED** [review]
worked for me, plus it's pretty cool :)

File: launcher/ui/instanceview/InstanceView.cpp [hunk 1]
```diff
@@ -498,12 +500,31 @@ void InstanceView::mouseDoubleClickEvent(QMouseEvent *event)
     }
 }
 
-void InstanceView::paintEvent(QPaintEvent *event)
+void InstanceView::setCatVisible(bool visible)
```
Comment (TheKodeToad) [inline launcher/ui/instanceview/InstanceView.cpp]: setPaintCat would be less confusing imo

File: launcher/ui/instanceview/InstanceView.cpp [hunk 2]
```diff
@@ -498,12 +500,31 @@ void InstanceView::mouseDoubleClickEvent(QMouseEvent *event)
...
+{
+    m_catVisible = visible;
+    m_catPixmap.load(QString(":/backgrounds/%1").arg(ThemeManager::getCatImage()));
```
Comment (TheKodeToad) [inline launcher/ui/instanceview/InstanceView.cpp]: I'm not sure if it'd work, but maybe if `visble` is `false` you could do `m_catPixmap = QPixmap()`

**Review by TheKodeToad — CHANGES_REQUESTED** [review]

**Review by TayouVR — APPROVED** [review]
seems to work and code looks good to me

**Review by TheKodeToad — APPROVED** [review]

**Pull request merged**